src/Gossip.js

```diff
@@ -244,10 +241,12 @@ export default class Gossip extends Component {
                                 </section>
                                 <div className="Gossip__Thread__Message__Content">
                                     {m.text}
-                                    {hasImage && <div className="Gossip__Thread__Message__Content__Att">
-                                        <img
-                                            src={m.meta.img_url}
-                                            onClick={() => this._handleAttachmentClick(m.meta.img_url)}/></div>}
+                                    {m.meta && <div className="Gossip__Thread__Message__Content__Att">
```

Comment:
We should also check if meta contains a URL so we don't render an empty Attachment div if meta is non-empty but URL is.